src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PersonList } from '../../components/PersonList/PersonList';
+
+export const PeoplePage = () => {
+  const [persons, setPersons] = useState<null | Person[]>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPersons)
+      .catch(() => setHasError(true))
```

Comment:
In the `catch` block, you should also catch a possible `error` and return it to the user or output it to the console:
```
.catch((error) => setHasError(`Something went wrong: ${error.message}`))
```